docs/sources/administration/configuration.md

```diff
@@ -292,7 +291,9 @@ The database user's password (not applicable for `sqlite3`). If the password con
 ### url
 
 Use either URL or the other fields below to configure the database
-Example: `mysql://user:secret@host:port/database`
+Example: `mysql://user:secret@
```

Comment:
Needs engineering input because this looks like code style.

Reply:
This is intended to be a URL, so must remain on one line.

Reply:
+1 to @hairyhenderson's comment.